boards/PDM/Inc/Io/Io_Efuse_Aux1Aux2.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
```

Comment:
I'd probs rename this to Io_Aux1Aux2Efuse.h. All the fxn names should also be renamed.

Reply:
Is that convention we use? _Io_Efuse_Aux1Aux1_ would be easier to do a search for and group together with the other Io_Efuse files I believe

Reply:
Yeah I think so. `Io_PrimaryScancon2RMHF` for example.